Source/utils/display.cpp

```diff
@@ -407,7 +408,7 @@ bool SpawnWindow(const char *lpWindowName)
 	SDL_SetHint(SDL_HINT_ORIENTATIONS, "LandscapeLeft LandscapeRight");
 #endif
 	if (SDL_Init(initFlags) <= -1) {
-		ErrSdl();
+		//ErrSdl();

```

Comment:
Is this still needed?

Reply:
It was there because sound init failed, but currently i just use NOSOUND because of other issues so that at least for now makes this not needed.

Reply:
Actually it looks like the SDL port is missing audio support. But lets wait with options for how to address this until the rest is in good condition. In worst case we could use something like Allegro just for audio.